src/network/networkd-network-gperf.gperf

```diff
@@ -178,6 +178,9 @@ IPv6Prefix.OnLink,                      config_parse_prefix_flags,
 IPv6Prefix.AddressAutoconfiguration,    config_parse_prefix_flags,                      0,                             0
 IPv6Prefix.ValidLifetimeSec,            config_parse_prefix_lifetime,                   0,                             0
 IPv6Prefix.PreferredLifetimeSec,        config_parse_prefix_lifetime,                   0,                             0
+CAN.BitRate,                            config_parse_si_size,                           0,                             offsetof(Network, can_bitrate)
+CAN.SamplePoint,                        config_parse_permille,                          0,                             offsetof(Network, can_sample_point)
+CAN.RestartSec,                         config_parse_sec,                               0,                             offsetof(Network, can_restart_us)
```

Comment:
Shouldn't this be called CAN.RestartSec? Reading this I was wondering how can I set times with sub second precision until I figured the usage.

Reply:
It _is_ called `CAN.RestartSec`. I grep'ed over the systemd source an all config options using the `config_parse_sec` parser end with `Sec`. However there is a `config_parse_nsec` parser and these options use `NSec`. The restart delay in the CAN world is usually some 100 micro seconds.

Reply:
The kernel interface is using microseconds, but IMHO that's not important here. I'll mention in the man page how set a timeout of 100ms.